presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -20,6 +20,7 @@
 import com.facebook.presto.spi.function.ScalarFunction;
 import com.facebook.presto.spi.function.SqlType;
 import com.facebook.presto.spi.type.StandardTypes;
+import com.facebook.presto.spi.type.TimeZoneKey;
```

Comment:
Fix "Fix at_timezone(TIME WITH TIME ZONE)" commit message

1. `s/Will yail/Will yield`
2. `s/This is done to use current offset of TZ as function that stucked in 1970-01-01 offsets seems useless./This is done because using time zone offsets differences as they were on 1970-01-01 seems useless.`